### ISSUE TITLE
Install error on Ubuntu 14.04

### ISSUE THREAD

**Issue by ajdroid** [clementfarabet/gfx.js#46]
> ==> installing static resources into ~/.gfx.js/
> ==> luajit not found, aborting... it must be in your path (this is a stupid installer)

`which luajit` says

> /usr/local/torch/install/bin/luajit

which makes sense since I installed torch for all users.

**Comment by soumith** [comment]
gfx.js itself is pretty depreceated. https://github.com/szym/display is more updated / maintained / works.

**Comment by ajdroid** [comment]
Yeah, just found your comments on the other issues. Will leave this open if someone decides to fix everything for some reason.
